core/vm/operations_verkle.go

```diff
@@ -158,7 +172,7 @@ func gasExtCodeCopyEIP4762(evm *EVM, contract *Contract, stack *Stack, mem *Memo
 		}
 		return gas, nil
 	}
-	wgas := evm.Accesses.TouchBasicData(addr[:], false, contract.Gas, true)
+	wgas := evm.Accesses.TouchBasicData(addr[:], false, contract.Gas-gas, true)
```

Comment:
We where using more gas than truly available.

Reply:
![image](https://github.com/user-attachments/assets/f9727e01-bac1-47a8-b59e-93d887d7a2cb)